### ISSUE TITLE
can you continue developing this note type

### ISSUE THREAD

**Issue by Vilhelm-Ian** [cjdduarte/anki-template-interactive-drag-drop#1]

**Comment by nidandre** [comment]
That would be great if we could drag and drop MP3s instead of text. Would that be possible ?

**Comment by nidandre** [comment]
I managed to add the MP3 : https://github.com/nidandre/Anki-Templates-for-Foreign-Language-Learning/tree/Sorting-with-Media

But it doesn't work with images.

**Comment by Vilhelm-Ian** [comment]
@nidandre use sort cards in my notetype it should work

**Comment by nidandre** [comment]
That's true. But I really like and prefer the design of this template : one table with different columns rather than seperate tables.
